sample/Kmd.Logic.Gateway.Automation.Sample/Publish/publish.yml

```diff
@@ -58,6 +60,7 @@ Apis:
       PolicyXmlFile: ./apis/nexus2/policies.xml
       BackendLocation: http://nexus2
       Status: Active
+      IsCurrent: true
```

Comment:
Why are we keeping both the versions Iscurrent = true, since it is a sample yaml it would be confusing for users. I think the latest version would be made as Iscurrent by the system. I think , lets set the first version to true and other version to false.

Reply:
Of course. My bad. I added this here, but was testing on other publish.yml file :D Thanks for noticing.